src/transformers/modeling_tf_utils.py

```diff
@@ -195,11 +195,21 @@ def hf_compute_loss(self, labels, logits):
         loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(
             from_logits=True, reduction=tf.keras.losses.Reduction.NONE
         )
+        if self.config.get("tf_legacy_loss", False):
```

Comment:
Configs are not dict, this will fail.
```suggestion
        if self.config.tf_legacy_loss:
```
(Same below)

Reply:
Ah, sorry, I meant to use `getattr`, but I guess the key will always be present anyway?